PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureDialog.java

```diff
@@ -57,23 +66,13 @@ public void close() {
     @Override
     protected List<ValidationInfo> doValidateAll() {
         final List<AzureValidationInfo> infos = this.getForm().getAllValidationInfos(true);
-        this.setOKActionEnabled(infos.stream().allMatch(AzureValidationInfo::isValid));
+        // this.setOKActionEnabled(infos.stream().allMatch(AzureValidationInfo::isValid));
```

Comment:
When shall we set Ok action enabled?

Reply:
intellij will handle the enabling/disabling status of OK button according to validation results.